### PR TITLE
GRW-2271 - refact(TabsBlock): updated to match latest designs

### PR DESCRIPTION
**Pull request by guilhermespopolin**
## Describe your changes

* Update TabsBlock to match latest designs and support [Support page implementation](https://www.figma.com/file/qUhLjrKl98PAzHov9ilaDH/*NEW*-.com-UI-Kit?node-id=1565%3A7541&t=NWIjsInKHcoEsoQf-4)

**Disclaimer**: Only the vertical Layout was implemented for now. We can iterate over it later

## Justify why they are needed

image

Can be tested [here](https://hedvig-dot-com-git-grw-2271-refacttabs-block-hedvig.vercel.app/se/hjalp/support-link-card)

## Jira issue(s): [GRW-2271](https://hedvig.atlassian.net/browse/GRW-2271)

[GRW-2271]: https://hedvig.atlassian.net/browse/GRW-2271?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by guilhermespopolin** [comment]
Current dependencies on/for this PR:
* main
  * **PR #1731** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1731" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1731?utm_source=stack-comment).

File: apps/store/src/blocks/TabsBlock.tsx [hunk 1]
```diff
@@ -50,7 +68,32 @@ const TabBlock = ({ blok }: TabBlockProps) => {
...
+export const Wrapper = styled.div({})
+
+const Title = styled(Heading)({
+  textAlign: 'center',
+  marginBottom: theme.space.lg,
+  paddingInline: theme.space.md,
+
+  [mq.lg]: {
+    marginBottom: theme.space.xl,
+  },
```
Comment (gustaveen) [inline apps/store/src/blocks/TabsBlock.tsx:83]: nit: `Heading` component has props for `margin` and `text-alignment`

File: apps/store/src/blocks/TabsBlock.tsx [hunk 2]
```diff
@@ -1,42 +1,60 @@
...
+        defaultValue={firstTabValue}
+        orientation={isLarge ? 'vertical' : 'horizontal'}
+      >
+        <GridLayout.Root>
+          <GridLayout.Content width="1/2" align="center">
+            <TabList>
+              {tabBlocks.map((tabBlock) => {
+                const tabId = tabBlock._uid || tabBlock.title
+                return (
+                  <Tabs.TabsTrigger key={tabId} value={tabId}>
```
Comment (gustaveen) [inline apps/store/src/blocks/TabsBlock.tsx:45]: nit: looks like they have a shadow and are a bit darker compared to design but feel free to omit if you talked with Zak about it 👍

File: apps/store/src/blocks/TabsBlock.tsx [hunk 3]
```diff
@@ -1,42 +1,60 @@
...
-    </Tabs.Tabs>
+    <Wrapper {...storyblokEditable(blok)}>
+      {blok.heading && (
+        <Title as="h2" variant={{ _: 'standard.24', lg: 'standard.32' }}>
+          {blok.heading}
+        </Title>
+      )}
+      <RadixTabs.Tabs
+        defaultValue={firstTabValue}
+        orientation={isLarge ? 'vertical' : 'horizontal'}
```
Comment (gustaveen) [inline apps/store/src/blocks/TabsBlock.tsx:37]: hm, is the layout supposed to change in `lg`, can't see any difference :D

**Review by gustaveen — APPROVED** [review]
Nice work! @guilhermespopolin 🎉

File: apps/store/src/blocks/TabsBlock.tsx (hunk 3 above)
Comment (guilhermespopolin) [reply apps/store/src/blocks/TabsBlock.tsx:37]: No, I forgot to remove that part. 👍🏻

**Pull request merged**